apps/web/src/components/layout/components/v2/SyncInfoModal.tsx

```diff
@@ -14,11 +14,12 @@ import { css } from '@novu/novui/css';
 export type SyncInfoModalProps = {
   isOpen: boolean;
   toggleOpen: () => void;
+  refetchOriginWorkflows: () => void;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Should this be async?
‏